### ISSUE TITLE
Индикация коннекта и переподключение коннекта (предложение)

### ISSUE THREAD

**Issue by AlTuzov** [SBProX/SBProX-Dev#854]
Думаю, было бы полезно в терминале видеть индикацию текущего подключения к серверу (выбранным коннектам). По скольку бывает сложно сразу заметить его отсутствие. На мой взгляд это хорошо сделано в квике. 

Также было бы полезно иметь возможность переподключаться к серверам без перезагрузки терминала. Например, при входе подключился к транзак-коннекту а в процессе работы переподключился к квику без перезагрузки терминала.

**Comment by Ravil124** [comment]
Возможность смены сервера без перезагрузки терминала, очень полезный функционал.

**Comment by flyserg** [comment]
Согласен, очень актуальный функционал, можно просто в виде цветного кружка как в Транзаке
